poppy/zernike.py

```diff
@@ -941,8 +948,8 @@ def __call__(self, nterms=None, npix=512, outside=np.nan):
         return basis[0:nterms]
 
 
-def opd_expand(opd, aperture=None, nterms=15, basis=zernike_basis,
-               **kwargs):
+def decompose_opd(opd, aperture=None, nterms=15, basis=zernike_basis,
+                  **kwargs):
     """Given a wavefront OPD map, return the list of coefficients in a
```

Comment:
The decompose_opd function's docstring (line 951 and 952) references `opd_expand_nonorthonormal` and `opd_expand_segments` when it should now reference "`ecompose_opd_nonorthonormal_basis` and `decompose_opd_segments`

Also, further down in the docstring on line 982, it says `opd_expand` when it should now say `decompose_opd`